impl/src/main/java/com/okta/sdk/impl/oauth2/OAuth2AccessToken.java

```diff
@@ -82,8 +82,8 @@ public boolean hasExpired() {
     }
 
     // for testing purposes
```

Comment:
if possible make it package-private if used for testing purposes

Reply:
yep, done!